components/service-operator/hack/test_integration.sh

```diff
@@ -8,8 +8,9 @@ docker build \
 	--build-arg AWS_ACCESS_KEY_ID \
 	--build-arg AWS_SECRET_ACCESS_KEY \
 	--build-arg AWS_SESSION_TOKEN \
-	--build-arg AWS_RDS_SECURITY_GROUP_ID=sg-04521d05ba3d9edb5 \
+	--build-arg AWS_RDS_SECURITY_GROUP_ID=sg-06ea6f076bd39a4f6 \
 	--build-arg AWS_RDS_SUBNET_GROUP_NAME=sandbox-private \
 	--build-arg AWS_PRINCIPAL_PERMISSIONS_BOUNDARY_ARN=arn:aws:iam::${AWS_ACCOUNT_ID}:policy/sandbox-service-operator-managed-role-permissions-boundary \
 	--build-arg AWS_PRINCIPAL_SERVER_ROLE_ARN=arn:aws:iam::${AWS_ACCOUNT_ID}:role/sandbox_kiam_server \
+	--build-arg AWS_ROLE_ARN=arn:aws:iam::${AWS_ACCOUNT_ID}:role/admin \
```

Comment:
shouldn't this be the service operator's role name?

Reply:
This script is for testing locally. When you run the integration tests locally the account used to execute this script becomes the service operator, which in our case is gsp-sandbox-admin.